pod/perldelta.pod

```diff
@@ -363,7 +363,7 @@ files in F<ext/> and F<lib/> are best summarized in L</Modules and Pragmata>.
 
 =item *
 
-XXX
+4-argument select now properly handles UTF8-flagged strings.
```

Comment:
C<select()>

Also, this doesn't really say what it does with those strings. What is the correct semantics here? I'm not sure offhand I can imagine what "properly" means in this case

Reply:
I think this is resolved with the new perldelta message.